plugins/package-managers/maven/src/main/kotlin/Tycho.kt

```diff
@@ -223,11 +227,56 @@ class Tycho(
             graphBuilder.addDependency(DependencyGraph.qualifyScope(projectId, node.dependency.scope), node)
         }
     }
+
+    /**
+     * Create a list with [Issue]s for global build problems. Since this implementation executes a single
+     * multi-module build, it is typically not possible to assign single issues to specific projects. Therefore, all
+     * issues are assigned to the root project. In order to generate the issues, evaluate the [exitCode] of the Maven
+     * build, the projects found by the [collector], and the [resolvedProjects] for which dependency information was
+     * found.
+     */
+    private fun createRootIssues(
+        exitCode: Int,
+        collector: TychoProjectsCollector,
+        resolvedProjects: List<MavenProject>
+    ): MutableList<Issue> {
+        val rootIssues = mutableListOf<Issue>()
+        if (exitCode != 0) {
+            rootIssues.add(
```

Comment:
Prefer the `+=` operator:

            rootIssues += createAndLogIssue(
                managerName,
                "No dependency information found for project '$coordinates'. " +
                    "This may be caused by a build failure."
            )

Also `Severity.ERROR` is the default an can be omitted.

Reply:
Similar below.